src/types/index.d.ts

```diff
@@ -0,0 +1,18 @@
+import internal from "stream"
+
+export interface Video {
+    video_id: string,
+    thumbnail: string,
+    title: string,
+    content_type: string,
+    url: string,
+    duration: internal,
```

Comment:
Internal doesnt seem right here, that should probably be a number?
```suggestion
    duration: number,
```

Reply:
Fixed.